### PR TITLE
Update teams-ip-phones.md

### PR DESCRIPTION
**Pull request by Leicham**
added a note for those looking to provision devices not currently certified by teams.

### CODE REVIEW

File: Teams/devices/teams-ip-phones.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ Beyond the certification period, Microsoft is committed to make efforts to suppo
 
 The following devices are Certified under the Microsoft Teams phones Certification Program:
 
+>![NOTE]
+>If your phone it not listed below, manually provisioning the phone may be possible via the device's Generic profile and as a SIP device. See [SIP Gateway Configure](https://docs.microsoft.com/en-us/microsoftteams/sip-gateway-configure).
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-ip-phones.md:42]:
```suggestion
> [!NOTE]
> If your phone is not listed below, manually provisioning the phone may be possible via the device's generic profile and as a SIP device. For more information, see [Configure SIP Gateway](/microsoftteams/sip-gateway-configure).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Leicham, kindly commit the suggested changes. Many thanks.

**Review by scanum — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
Change declined by product team.

**Pull request closed without merging**